apps/src/studio/blocks.js

```diff
@@ -1769,11 +1777,11 @@ exports.install = function(blockly, blockInstallOptions) {
       var rows = [
         'title, subtitle, background',
         [this.blockArgs[0], this.blockArgs[1], this.blockArgs[2]],
-        'danger, target, player',
+        'target, danger, player',
         [this.blockArgs[3], this.blockArgs[4], this.blockArgs[5]],
         'update-target, update-danger, update-player',
         [this.blockArgs[6], this.blockArgs[7], this.blockArgs[8]],
-        'collide?, on-screen?',
+        'collide?, onscreen?',
```

Comment:
I only changed it to 'onscreen?' in the title, because when I changed the argument name (here and in BigGameLogic) I kept getting Uncaught Input does not exist: on-screen?